runtime/commonMain/src/kotlinx/serialization/SealedSerializer.kt

```diff
@@ -84,7 +91,8 @@ public class SealedClassSerializer<T : Any>(
     init {
         require(subclasses.size == subclassSerializers.size) {
             "Arrays of classes and serializers must have the same length," +
-                    " got arrays: ${subclasses.contentToString()}, ${subclassSerializers.contentToString()}"
+                    " got arrays: ${subclasses.contentToString()}, ${subclassSerializers.contentToString()}\n" +
+                    "Please ensure that @Serializable annotation is present on each sealed subclass"
```

Comment:
Plugin automatically filters out non-serialziable subclasses, afair. So this error may happen only on manual construction of sealed serializer.